massgov/mayflower-patternlab
# Add contact row organism

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

This PR adds a new 'contact row' organism to alleviate some issues with the contact list organism being used in certain new contexts.

## Related Issue / Ticket

https://jira.state.ma.us/browse/DP-8993
https://jira.state.ma.us/browse/DP-8992
https://jira.state.ma.us/browse/DP-8969


## Steps to Test


1.  View component is isolation http://localhost:3000/?p=organisms-contact-row
2. and variations:
    -  http://localhost:3000/?p=organisms-contact-row-no-primary
    - http://localhost:3000/?p=organisms-contact-row-single-additional-contact
    - http://localhost:3000/?p=organisms-contact-row-single-contact

3. Confirm that spacing and positioning issues from the following are not present:
- https://jira.state.ma.us/browse/DP-8993
- https://jira.state.ma.us/browse/DP-8992
- https://jira.state.ma.us/browse/DP-8969
4. On the following pages:
- http://localhost:3000/?p=pages-organization-elected-official
- http://localhost:3000/?p=pages-organization-boards
- http://localhost:3000/?p=pages-organization


## Screenshots



## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* new organism 'contact-row'
